### DISCUSSION TITLE
Is it necessary to create a team to deploy a repository in an organization?

### DISCUSSION THREAD

**Discussion by miugel** [vercel/community#39] [Help] [answered]
Other hosting platforms provide the option to deploy a repository in an organization. I'm not seeing the option to do that with Zeit unless I create a team, which is a paid feature.

**Comment by wiiiimm** [comment, marked as answer]
Hi @miugel, you are right. On the free hobby plan, you can deploy from only from GitHub / GitLab / Bitbucket personal accounts. Deploying from GitHub Organisations / GitLab Groups / Bitbucket Teams is limited to paid plans but we do offer 14 days trials on new teams.

Aside from that, we do also offer OSS sponsorships if you are working on open source / charity / education related projects, you can visit https://vercel.com/support/articles/can-vercel-sponsor-my-open-source-project for more info.

Lastly, you can also create manual deployments using Vercel CLI but you will miss out a lot of cool Git integration features.

Please check out https://vercel.com/pricing for details.

**Reply by imabp** [reply]
Hello @williamli . I have raised a query for OSS sponsorship to `sponsorships@vercel.com` as mentioned here
https://vercel.com/support/articles/can-vercel-sponsor-my-open-source-project

I got a confirmation, that within 5 days, I will be getting response, now it's more than a week.
Can you please help me out. If you want to know the case number, do let me know.

**Reply by AIEngKenn** [reply]
I'd like to know more about it. Did it actually work?

**Reply by saad-devx** [reply]
Does someone know about this?? i am just importing a Bit Bucket repository to deploy on hobby plan but vercel asks me to add payment card saying: "Because you will be deploying from a Bitbucket workspace, creating a Vercel Team is required for collaborating with others. You've already used your free trial, so you will need to add a credit card to create a team." I just wan to import and deploy a simple project what should i do please help!

**Comment by MathiasReker** [comment]
It is free to create an organization, you just have to be at least 3 members.
